xgi/classes/hypergraph.py

```diff
@@ -513,20 +513,21 @@ def add_edges_from(self, ebunch_to_add, **attr):
 
         Parameters
         ----------
-        ebunch_to_add : iterable
+        ebunch_to_add : Iterable
 
             An iterable of edges, in one of the following formats.
 
-            * Format 1: Iterable of hashables (id1, id2, ...), or
+            * Format 1: Iterable (edge1_members, edge2_members), or
```

Comment:
Wait, this should a single edge, so this is now wrong?
Should be (node_id1, node_id1, ...) or (edge_members) I guess.

Reply:
It was wrong before, I fixed it now.